### ISSUE TITLE
NumberFormatException: Invalid int: "4.7.0"

### ISSUE THREAD

**Issue by dss886** [airbnb/lottie-android#488]
I have 4 lottie animation in my app, and the "4.7.0" string is in the beginning of every json file:

{
    "v":"4.7.0",
    ......
}

After release, I got a crash (only once) on a specific device named "MI PAD 2". 
I guess that "v" may means the version of the AE plugin that output the json file.
So I don't know how this happened, should I just ignore it?

Full stacktrace:

~~~java
java.lang.RuntimeException: An error occured while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:304)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NumberFormatException: Invalid int: "4.7.0"
	at java.lang.Integer.invalidInt(Integer.java:138)
	at java.lang.Integer.parse(Integer.java:410)
	at java.lang.Integer.parseInt(Integer.java:367)
	at java.lang.Integer.parseInt(Integer.java:334)
	at com.airbnb.lottie.LottieComposition$Factory.fromJsonSync(LottieComposition.java:271)
	at com.airbnb.lottie.LottieComposition$Factory.fromInputStream(LottieComposition.java:241)
	at com.airbnb.lottie.model.FileCompositionLoader.doInBackground(FileCompositionLoader.java:20)
	at com.airbnb.lottie.model.FileCompositionLoader.doInBackground(FileCompositionLoader.java:10)
	at android.os.AsyncTask$2.call(AsyncTask.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 3 more
java.lang.NumberFormatException: Invalid int: "4.7.0"
	at java.lang.Integer.invalidInt(Integer.java:138)
	at java.lang.Integer.parse(Integer.java:410)
	at java.lang.Integer.parseInt(Integer.java:367)
	at java.lang.Integer.parseInt(Integer.java:334)
	at com.airbnb.lottie.LottieComposition$Factory.fromJsonSync(LottieComposition.java:271)
	at com.airbnb.lottie.LottieComposition$Factory.fromInputStream(LottieComposition.java:241)
	at com.airbnb.lottie.model.FileCompositionLoader.doInBackground(FileCompositionLoader.java:20)
	at com.airbnb.lottie.model.FileCompositionLoader.doInBackground(FileCompositionLoader.java:10)
	at android.os.AsyncTask$2.call(AsyncTask.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
~~~

I have seen the issue of #422, but can't offer any help.

**Comment by gpeal** [comment]
@dss886 if you only saw this one time on one device, I'd ignore it. If this starts happening to more users on more devices, feel free to reopen and attach the animation that crashes.

**Comment by huangyanan** [comment]
i also have this crash on the device named "MI PAD 2".  it now only happen 3 times on my crash analyse tool .

**Comment by inemnitable** [comment]
I'm seeing a fair number of these crashes (~hundreds) in production on several devices. The affected devices include Hudl 2, ASUS MeMO Pad 7, Hi10 plus, and quite a few others. I'm only seeing crashes on OS version 5.1 and 5.1.1, but I'm not sure that that's related.

I dug a bit into the code and the only explanation I could come up with is that these few devices suffer from an incorrect implementation of String.split().

**Comment by gpeal** [comment]
This is a duplicate of #559

**Comment by 103style** [comment]
java.lang.NumberFormatException
Invalid int: "4.9.0"
com.airbnb.lottie.e$a.fromJsonSync(Proguard)

I got the same crash  three times on Nokia N1

**Comment by gpeal** [comment]
@103style What version of lottie?
